docs/developer_guide/dependencies.md

```diff
@@ -47,7 +47,7 @@ require (
 )
 ```
 Release branches **must** not use pseudo versions.
-During the creation of a release, [tags](/community/release_procedure#tag-and-update-release-for-version) will be created.
+During the creation of a release, [tags](../../src/pages/community/release_procedure.mdrelease_procedure#tag-and-update-release-for-version) will be created.
```

Comment:
the website build converts the links, this looks broken.
`local_build.sh run` will build and run the site on your local machine to check

Reply:
You are right.  I tested this locally and I thought it was a bad link.  Didn't know about the website build.  I'll check the main webpage for broken links for now on.